openfl-tutorials/Federated_Keras_MNIST_Tutorial.ipynb

```diff
@@ -14,7 +14,7 @@
    "outputs": [],
    "source": [
     "#Install dependencies if not already installed\n",
-    "!pip install tensorflow mnist"
+    "!pip install tensorflow==2.3 mnist"
```

Comment:
@kshannon  I would recommend to use 2.3.1 version to be consistent with [workspace requirements](https://github.com/intel/openfl/blob/develop/openfl-workspace/keras_cnn_mnist/requirements.txt#L1)
And since you are adding check for intel-tensorflow, let's add some comment like that:
```
"!pip install tensorflow==2.3.1 mnist"
# Or you could use intel-tensorflow
# !pip install intel-tensorflow==2.3.0 mnist"
```